core/tests/ts-integration/contracts/custom-account/custom-account.sol

```diff
@@ -9,6 +9,10 @@ import './SystemContractsCaller.sol';
 
 import './interfaces/IAccount.sol';
 
+interface ITimestampAsserter {
+	function assertTimestampInRange(uint256 start, uint256 end) external view;
```

Comment:
Bikeshedding: I think (fortunately), tab indentation isn't usually used in Solidity. Would be somewhat to change in this case though.

Reply:
Do you mean changing TAB to 4 spaces?